flutter_appauth/example/lib/main.dart

```diff
@@ -182,11 +188,11 @@ class _MyAppState extends State<MyApp> {
       _setBusyState();
       final TokenResponse? result = await _appAuth.token(TokenRequest(
           _clientId, _redirectUrl,
-          refreshToken: _refreshToken, issuer: _issuer, scopes: _scopes));
+          refreshToken: _refreshToken! + "a", issuer: _issuer, scopes: _scopes));
```

Comment:
I suspect this was an unintentional change (?)

Reply:
Yes 🤦 . Thought I'd removed that since making a PR, it was the easiest way to check  the errors were coming through properly